change/@fluentui-react-provider-d5d6ba04-fdeb-4003-be12-17ef140ed50c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
you are adding new functionality - this should be minor/feature

Reply:
I generated another change file for that.